### PR TITLE
Add config profile option

### PR DESCRIPTION
**Pull request by mdellweg**
This is a request for comment.
Do we want something like this?

### CODE REVIEW

**Comment by daviddavis** [comment]
This looks good to me. :+1:

**Comment by mdellweg** [comment]
I'm going to add some tests...

**Comment by mdellweg** [comment]
I moved some tests i think are not testing pulpcore in any reasonable way out of the pulpcore test directory.
@daviddavis is this OK for you?

File: tests/scripts/test_config.sh [hunk 1]
```diff
@@ -29,6 +30,14 @@ expect_fail pulp --username test --password test --client "/some/path" status
 expect_fail pulp --key "/some/path" file remote list
 
 
+# CONFFIG PROFILE
```
Comment (daviddavis) [inline tests/scripts/test_config.sh:33]:
```suggestion
# CONFIG PROFILE
```

**Review by daviddavis — APPROVED** [review]

File: tests/scripts/test_config.sh (hunk 1 above)
Comment (mdellweg) [reply tests/scripts/test_config.sh:33]: Hmm this keyboard of mine is really oversensitive...

**Pull request merged**